cedar-policy-validator/src/human_schema/ast.rs

```diff
@@ -95,13 +95,8 @@ impl Path {
     }
 
     /// Is this referring to a name in the `__cedar` namespace (eg: `__cedar::Bool`)
```

Comment:
```suggestion
    /// Is this referring to a name in the `__cedar` namespace (eg: `__cedar::Bool`)
    /// and/or otherwise reserved (e.g., `A::__cedar::Bool`)
```